### PR TITLE
product module api test cases and code issue resolved

### PR DESCRIPTION
**Pull request by darkl1ght**
Hi @ageddesi, Kindly review this PR for the requested issue #123.

### CODE REVIEW

File: modules/products/api/products-routes.ts [hunk 1]
```diff
@@ -156,7 +157,7 @@ module.exports = function (app: core.Express) {
 	 *         schema:
 	 *           $ref: '#/definitions/MockProduct'
 	 */
-	app.get('/products/:qty?', (req: Request, res: Response) => {
+	app.get('/products/quantity/:qty?', (req: Request, res: Response) => {
```
Comment (ageddesi) [inline modules/products/api/products-routes.ts:160]: I do not feel we need /quantity here however I do like the use of /deparment above :)

File: utils/swagger.ts [hunk 2]
```diff
@@ -59,6 +59,10 @@ const options: swaggerJsdoc.Options = {
                 name: 'Images',
                 description: 'A set of endpoints related to obtaining placeholder image urls',
             },
+            {
```
Comment (ageddesi) [inline utils/swagger.ts:62]: Can you just confirm this is in the newer place of swagger-config under utils. We do not directly update the swagger docs anymore.

Comment (darkl1ght) [reply utils/swagger.ts:62]: Hi, I added it here seeing the tags that are present in the swagger page. So, yes this is where we can add the tags for the swagger page.

File: modules/products/api/products-routes.ts (hunk 1 above)
Comment (darkl1ght) [reply modules/products/api/products-routes.ts]: @ageddesi I have removed and PR should be updated.

**Review by ageddesi — DISMISSED** [review]

**Comment by ageddesi** [comment]
Hi @darkl1ght Just seems to be a couple of conflicts. If you can resolve them please this will get pulled in :)

**Review by darkl1ght — COMMENTED** [review]
@ageddesi Merge Issues have been resolved. Please review and approve the changes.

**Review by ageddesi — APPROVED** [review]

**Comment by ageddesi** [comment]
Hi @darkl1ght , thank you for the hard work on this 👍 Happy hacktoberfest :)

**Comment by darkl1ght** [comment]
hi @ageddesi I am getting not-accepted status on my hacktober fest page

image

Could you please check this once?

**Pull request merged**